shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -130,15 +142,16 @@ @implementation FlutterTextInputPlugin {
 - (instancetype)initWithViewController:(FlutterViewController*)viewController {
   self = [super init];
   if (self != nil) {
-    _flutterViewController = viewController;
```

Comment:
Why do we remove the _flutterViewController? if we do want to remove it, we should remove all the reference to it and the private property as well. cc @dkwingsmt

Reply:
The reason I removed it is because it's not used after initializing `_channel`.

It's indeed an oversight for me not to remove the private property.

Reply:
A later commit reused flutterViewController https://github.com/flutter/engine/pull/24867, I think we should add it back

Reply:
Sure. Sorry for the confusion I caused.
Although I see https://github.com/flutter/engine/pull/25524 is pending to fix the caret, can you just ask him to include this change?

Reply:
will do :)